plugins/modules/ndo_l3out_floating_svi_interface_path_attributes.py

```diff
@@ -0,0 +1,375 @@
+    default: query
+notes:
+- The O(template) or O(template_id) must exist before using this module in your playbook.
+  The M(cisco.mso.ndo_template) module can be used for this.
+- The O(l3out) or O(l3out_uuid) must exist before using this module in your playbook.
+  The M(cisco.mso.ndo_l3out_template) module can be used for this.
+- The L3Out Floating SVI Interface identified by O(node_id), O(encapsulation_type), and O(encapsulation_value)
+  must exist before using this module in your playbook.
+  The M(cisco.mso.ndo_l3out_floating_svi_interface) module can be used for this.
+seealso:
+- module: cisco.mso.ndo_template
+- module: cisco.mso.ndo_l3out_template
+- module: cisco.mso.ndo_l3out_floating_svi_interface
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create Path Attributes for a L3Out Floating SVI Interface
+  cisco.mso.ndo_l3out_floating_svi_interface_path_attributes:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: l3out_template
+    l3out: l3out_name
+    node_id: 101
+    encapsulation_type: vlan
+    encapsulation_value: 1200
+    domain_type: vmm
+    domain_provider: vmware
+    domain: domain_name
+    forged_transmit: true
+    mac_address_change: true
+    promiscuous_mode: true
+    enhanced_lag_policy: enhanced_lag_policy_name
+    primary_ipv4_address: 192.0.2.1
+    primary_ipv6_address: 2001:db8::1
+    state: present
+
+- name: Update the Path Attributes of a L3Out Floating SVI Interface
+  cisco.mso.ndo_l3out_floating_svi_interface_path_attributes:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template_id: "{{ l3out_template.current.templateId }}"
+    l3out_uuid: "{{ l3out.current.uuid }}"
+    node_id: 101
+    encapsulation_type: vlan
+    encapsulation_value: 1200
+    domain_type: vmm
+    domain_provider: vmware
+    domain: domain_name
+    forged_transmit: false
+    mac_address_change: false
+    promiscuous_mode: false
+    enhanced_lag_policy: enhanced_lag_policy_name
+    primary_ipv4_address: 192.0.2.2
+    primary_ipv6_address: 2001:db8::2
+    state: present
+
+- name: Query the Path Attributes of an existing L3Out Floating SVI Interface
+  cisco.mso.ndo_l3out_floating_svi_interface_path_attributes:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: l3out_template
+    l3out: l3out_name
+    node_id: 101
+    encapsulation_type: vlan
+    encapsulation_value: 1200
+    domain_type: vmm
+    domain_provider: vmware
+    domain: domain_name
+    state: query
+  register: query_one
+
+- name: Query all existing Path Attributes of L3Out Floating SVI Interface
+  cisco.mso.ndo_l3out_floating_svi_interface_path_attributes:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: l3out_template
+    l3out: l3out_name
+    node_id: 101
+    encapsulation_type: vlan
+    encapsulation_value: 1200
```

Comment:
are those really required for query?

Reply:
yes because the encap combination (type/value) is used while identifying the floating svi ( parent object ) of path attributes